fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/visitor/ExpressionVisitor.java

```diff
@@ -505,6 +506,10 @@ public R visitMatchPhraseEdge(MatchPhraseEdge matchPhraseEdge, C context) {
         return visitMatch(matchPhraseEdge, context);
     }
 
+    public R visitPlaceholderExpr(PlaceholderExpr placeholderExpr, C context) {
+        return null;
```

Comment:
should not return null. return `visit(placeholderExpr, context)`

Reply:
done